rebar.config

```diff
@@ -7,6 +7,7 @@
 {profiles, [{test, [{erl_opts, [nowarn_export_all]},
                     {deps, [jsone,
                             opentelemetry,
+                            opentelemetry_api,
```

Comment:
https://github.com/erleans/pgo/pull/85#issuecomment-2313929256 <- this fixed this error. The applications list has this, but I guess including the dependency fetches the include files? Sorry, I am not actually an erlang expert.

Reply:
Hm, weird, this shouldn't be needed since it is included in the default profile deps.